service/synapse-service-rest/src/main/java/io/americanexpress/synapse/service/rest/controller/reactive/helpers/ReactiveCreateResponseEntityCreator.java

```diff
@@ -38,7 +38,7 @@ public ResponseEntity<Mono<O>> create(O serviceResponse) {
         String responseIdentifier = "0";
```

Comment:
here also can we change to responseId, just to keep it consistent.

Reply:
i concur

Reply:
Done